### DISCUSSION TITLE
Qiita記事のURLを貼ってもサムネイルやタイトルが出ないものがある

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#921] [要望・アイデア・不具合]
https://qiita.com/jnchito/items/ec8d625c83b207f21ded という記事に、 https://qiita.com/jnchito/items/91779b55cae0f4915da5 を貼り付けたのですが、サムネイルやタイトルが出ず添付画像のように表示されてしまいます。
何度やり直しても同じ結果でした。

![Screenshot 2024-10-28 at 8 29 08](https://github.com/user-attachments/assets/49fe6a50-5188-4ef6-9bfd-c48ab923c0e8)

これは何が原因なのでしょうか？

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ご不便をおかけして、申し訳ございません。

Qiita記事でのOGP表示にはキャッシュを利用していることが原因のようです。
弊社でキャッシュクリアをしましたので、ご確認をいただけますと幸いです🙇

**Comment by JunichiIto** [comment]
修正されているのを確認しました。
ご対応ありがとうございます！
